app/views/content.js

```diff
@@ -0,0 +1,35 @@
+import Mn from 'backbone.marionette';
+import HeaderView from './header';
+import TracksView from './tracks';
+import application from '../application';
+
+
+const Content = Mn.LayoutView.extend({
+
+    template: require('./templates/content'),
+
+    regions: {
+        header: '[role="complementary"]',
+        tracks: '#tracks',
+    },
+
+    onBeforeShow() {
+        let header = new HeaderView({ model: application.appState })
+        this.showChildView('header', header);
+        this.listenTo(
+            application.appState,
+            'change:currentPlaylist',
+            this.switchPlaylist
+        );
+        this.switchPlaylist();
```

Comment:
To handles the callback form (see below), call it with `this.switchPlaylist(null, application.appState.get('currentPlaylist'))`